<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>70</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
	font: 11pt &quot;楷体&quot;;
	background-color: rgb(85, 255, 127);
	
				   border-radius:10px;
				   
                                                                
                   }

                   QPushButton:pressed{
                   
	
	
                   }
                   
                   QPushButton:hover{
background-color: rgb(0, 85, 255);
                   }</string>
   </property>
   <property name="text">
    <string>查看账单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>70</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
	font: 11pt &quot;楷体&quot;;
	background-color: rgb(85, 255, 127);
	
				   border-radius:10px;
				   
                                                                
                   }

                   QPushButton:pressed{
                   
	
                   }
                   
                   QPushButton:hover{
background-color: rgb(0, 85, 255);
                   }</string>
   </property>
   <property name="text">
    <string>打印账单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>70</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
	font: 11pt &quot;楷体&quot;;
	background-color: rgb(85, 255, 127);
	
				   border-radius:10px;
				   
                                                                
                   }

                   QPushButton:pressed{
                   
	
                   }
                   
                   QPushButton:hover{
background-color: rgb(0, 85, 255);
                   }</string>
   </property>
   <property name="text">
    <string>查看详单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>70</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
	font: 11pt &quot;楷体&quot;;
	background-color: rgb(85, 255, 127);
	
				   border-radius:10px;
				   
                                                                
                   }

                   QPushButton:pressed{
                   
	
                   }
                   
                   QPushButton:hover{
background-color: rgb(0, 85, 255);
                   }</string>
   </property>
   <property name="text">
    <string>打印详单</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 11pt &quot;楷体&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string> 房间号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
font: 11pt &quot;楷体&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
	font: 11pt &quot;楷体&quot;;
	background-color: rgb(85, 255, 127);
	
				   border-radius:10px;
				   
                                                                
                   }

                   QPushButton:pressed{
                   
	
                   }
                   
                   QPushButton:hover{
					
	background-color: rgb(0, 85, 255);
                   }</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>471</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0.5);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
